### ISSUE TITLE
Latest prerelease version stopped working

### ISSUE THREAD

**Issue by genodeftest** [RequestPolicyContinued/requestpolicy#862]
Read first: https://requestpolicycontinued.github.io/Contributing

__Browser and Add-ons:__

- Browser: Firefox 56.0 (firefox-56.0-5.fc26.x86_64)
- RequestPolicy version: 1.0.beta13.2.1876.r489ddf49.pre
- Other add-ons:
  - CanvasBlocker-Beta
  - ConsistentHTTPS
  - FireGestures
  - google-no-tracking-url
  - HTTPS Everywhere
  - NoScript
  - Smart Referer
  - Tab Mix Plus
  - uBlock Origin

RequestPolicy settings:
- Default policy: "deny"

__Steps to reproduce:__
1. Open my old Firefox profile.

__What happens?__
Today, when I started Firefox, I noticed that the RequestPolicyContinued icon is missing and all content configured to be blocked by RPC is not being blocked. Both, `about:support` and `about:addons` show the addon as enabled. This is from my syslog:

```
console.log: [RequestPolicy] starting up
console.error:
  [RequestPolicy] Loader.main() failed!
console.dir:
  Message: Module `content/main.js` is not found at chrome://rpcontinued/content/content/main.js
  Stack:
    load@chrome://rpcontinued/content/bootstrap/bootstrap.jsm:110:16
FakeWebExt.startup@chrome://rpcontinued/content/bootstrap/bootstrap.jsm:198:5
startup@jar:file:///home/username1/.mozilla/firefox/cycswd8m.main2/extensions/rpcontinued@amo.requestpolicy.org.xpi!/bootstrap.js:55:3
callBootstrapMethod@resource://gre/modules/addons/XPIProvider.jsm:4418:11
startup@resource://gre/modules/addons/XPIProvider.jsm:2244:13
callProvider@resource://gre/modules/AddonManager.jsm:269:12
_startProvider@resource://gre/modules/AddonManager.jsm:739:5
startup@resource://gre/modules/AddonManager.jsm:906:9
startup@resource://gre/modules/AddonManager.jsm:3090:5
observe@jar:file:///usr/lib64/firefox/omni.ja!/components/addonManager.js:65:9
1508581537100        addons.xpi        WARN        Exception running bootstrap method startup on rpcontinued@amo.requestpolicy.org: Module `content/main.js` is not found at chrome://rpcontinued/content/content/main.js (chrome://rpcontinued/content/bootstrap/bootstrap.jsm:110) JS Stack trace: load@bootstrap.jsm:110:16 < FakeWebExt.startup@bootstrap.jsm:198:5 < startup@jar:file:///home/username1/.mozilla/firefox/cycswd8m.main2/extensions/rpcontinued@bootstrap.js:55:3 < callBootstrapMethod@XPIProvider.jsm:4418:11 < startup@XPIProvider.jsm:2244:13 < callProvider@AddonManager.jsm:269:12 < _startProvider@AddonManager.jsm:739:5 < startup@AddonManager.jsm:906:9 < startup@AddonManager.jsm:3090:5 < observe@addonManager.js:65:9
```

__What should happen?__
Start RequestPolicyContinued.

__Additional information:__
The previous prerelease announced on [Sept 18](https://github.com/RequestPolicyContinued/requestpolicy/issues/704#issuecomment-330342528) worked fine.

**Comment by genodeftest** [comment]
The latest released version (1.0.beta13.2) works fine though.

**Comment by myrdd** [comment]
I guess I'm going to revert commit b7c6086c55c4df0497e2f106bf3841f7014df21b, just to be sure.
